client/document/500.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Head from '../components/head';
+
+class ServerError extends Component {
+	render() {
+		const { urls, faviconURL } = this.props;
+		return (
+			<html lang="en">
+				<Head faviconURL={ faviconURL } cdn={ '//s1.wp.com' }>
+					<link rel="stylesheet" id="main-css" href={ urls[ 'style.css' ] } type="text/css" />
+				</Head>
+				<body>
+					{ /* eslint-disable wpcalypso/jsx-classname-namespace*/ }
+					<div id="wpcom" className="wpcom-site">
+						<div className="wp has-no-sidebar">
+							<div className="layout__content" id="content">
+								<div className="layout__primary" id="primary">
+									<div className="empty-content">
```

Comment:
Let's reuse the [`<EmptyContent />`](https://github.com/Automattic/wp-calypso/tree/master/client/components/empty-content) component here :slightly_smiling_face: